js/count-letter-in-string.js

```diff
@@ -0,0 +1,15 @@
+// https://www.codewars.com/kata/52efefcbcdf57161d4000091/train/javascript
+
+function count(string) {
+  const lettersCount = {};
+  for (let i = 0; i < string.length; i++) {
```

Comment:
`for` works fine, but is imperative :smile: Could you use a more declarative solution, e.g. `forEach`